src/main/java/model/FruitTransaction.java

```diff
@@ -0,0 +1,53 @@
+package model;
+
+public class FruitTransaction {

```

Comment:
According to the checklist, there should not be an empty line at the beginning of class definitions. Please remove the empty line to comply with the coding standards.